### ISSUE TITLE
How do I use the gem with Mongoid? [ANSWERED]

### ISSUE THREAD

**Issue by dks17** [lynndylanhurley/devise_token_auth#1263]
Until new version of the gem will be released you should use origin repository (not fork) and `master` branch of it.

**Gemfile**
```ruby
gem 'mongoid', '7.0'
gem 'mongoid-locker', '~> 1.0'

gem 'devise_token_auth', github: 'lynndylanhurley/devise_token_auth', branch: 'master'
```
To create user model for Mongoid
```ruby
rails g devise_token_auth:install_mongoid User auth
```

For further, please read the [Docs](https://devise-token-auth.gitbook.io/devise-token-auth/config).

**Comment by bjcCin** [comment]
If you already installed devise_token_auth, you will need uninstall before execute "devise_token_auth:install_mongoid" command.

**Comment by MaicolBen** [comment]
I think this can be closed now as Mongo is in v1.1.0
